book/modules/appendix3-exercises.tex

```diff
@@ -38,15 +56,63 @@
 
 			\item $\Set*{\mat{4 \\ 2 \\ 4},\mat{4 \\ 2 \\ 0},\mat{2 \\ 1 \\ 6}}$
 		\end{enumerate}
-
-		\prob Find all values of $a,b\in \R$ so that the set
-			$\Set*{\mat{a^2 \\ ab},\mat{ab \\ b^2}}$ is (a)
-			a right-handed basis, (b) a left-handed basis, (c) not a basis.
+        \begin{solution}
+        \begin{enumerate}
+	        \item Since $\det(\mat{1 & 1 \\ -3 & 2})=5>0$, the ordered set
+		        $\Set*{\mat{1 \\ -3},\mat{1 \\ 2}}$ is a right-handed basis.
+
+	        \item Since $\det(\mat{1 & -2 \\ -3 & 6})=0$, the set
+		        $\Set*{\mat{1 \\ -3},\mat{-2 \\ 6}}$ is not linearly independent and so
+		        is not a basis.
+
+	        \item Since $\det(\mat{1 & 1 & 1 \\ 0 & 2 & -1 \\ 3 & 5 & 1})=-2<0$, the
+		        ordered set
+		        $\Set*{\mat{1 \\ 0 \\ 3},\mat{1 \\ 2 \\ 5},\mat{1 \\ -1 \\ 1}}$ is a
+		        left-handed basis.
+
+	        \item Since $\det(\mat{1 & 1 & 1 \\ 4 & 2 & 1 \\ 9 & 3 & 1})=-2<0$, the
+		        ordered set
+		        $\Set*{\mat{1 \\ 4 \\ 9},\mat{1 \\ 2 \\ 3},\mat{1 \\ 1 \\ 1}}$ is a left-handed
+		        basis.
+
+	        \item Since $\det(\mat{4 & 4 & 2 \\ 2 & 2 & 1 \\ 4 & 0 & 6})=0$, the set
+		        $\Set*{\mat{4 \\ 2 \\ 4},\mat{4 \\ 2 \\ 0},\mat{2 \\ 1 \\ 6}}$ is not
+		        linearly independent and so is not a basis.
+        \end{enumerate}
+        \end{solution}
+
+        \prob Find all values of $a,b\in \R$ so that the set
+        $\Set*{\mat{a^2 \\ ab},\mat{ab \\ b}}$ is (a) a right-handed basis, (b) a left-handed
+        basis, (c) not a basis.
+        \begin{solution}
+	        Note that $\det\left(\mat{a^2 & ab \\ ab & b}\right)=a^{2}b-a^{2}b^{2}=a^{2}(
+	        b-b^{2})$.
+	        \begin{itemize}
+		        \item If $\det\left(\mat{a^2 & ab \\ ab & b}\right)=0$, then either
+			        $a^{2}=0$ or $b-b^{2}=0$. If $a^{2}=0$, then $a=0$. If $b-b^{2}=0$,
+			        then $b=0$ or $b=1$.
+
+		        \item If $\det\left(\mat{a^2 & ab \\ ab & b}\right)>0$, then $a^{2}$ and
+			        $b-b^{2}$ have the same sign. Since $a^{2}>0$, we must have
+			        $b-b^{2}>0$. Then $0<b<1$.
+
+		        \item If $\det\left(\mat{a^2 & ab \\ ab & b}\right)<0$, then $a^{2}$ and
+			        $b-b^{2}$ have different signs. Since $a^{2}>0$, we must have
+			        $b-b^{2}<0$. Then $b<0$ or $b>1$.
+	        \end{itemize}
+	        To sum up, $\Set*{\mat{a^2 \\ ab},\mat{ab \\ b}}$ is a right-handed basis when
+	        $a\ne 0$ and $0<b<1$; $\Set*{\mat{a^2 \\ ab},\mat{ab \\ b}}$ is a left-handed
+	        basis when $a\ne 0$ and either $b<0$ or $b>1$; $\Set*{\mat{a^2 \\ ab},\mat{ab \\ b}}$
+	        is not a basis when $a=0$ or $b=0$ or $b=1$.
```

Comment:
This needs to be reworked. The question refers to parts (a), (b), and (c). So the answer should name those parts explicitly. If you need to do pre-work, say something like, "Before answering, we compute ...."